addons/web/static/src/core/position_hook.js

```diff
@@ -310,7 +310,7 @@ export function usePosition(target, options) {
     const update = () => {
         const targetEl = getTarget();
         const popperEl = popperRef.el;
-        if (!targetEl || !popperEl) {
+        if (!targetEl || !popperEl || !targetEl.ownerDocument.contains(targetEl)) {
```

Comment:
Sorry for the spam, forgot to re-check this.

If we avoid calling it when "fixedPosition" is specified, do we still need this fix?

Reply:
for properties no, but it might occurs somewhere else in Odoo

Do you prefer to not fix it here ?

Reply:
ok, I removed both fix so